components/02-molecules/menu/menu.twig

```diff
@@ -58,7 +62,7 @@
 
 <nav {{ add_attributes(menu__attributes) }}>
```

Comment:
@kara-franco Accessibility question for you. Is it acceptable to use `aria-label` in this instance on the `<nav>` or is it best to use `aria-labelledby="nav-heading"` followed by an `<h2 id="nav-heading">My awesome nav</h2>`

Reply:
My understanding on this is that if the "label" (in your example the `h2`) already exists, use `aria-labelledby` and point to it. If it doesn't already exist, use `aria-label`. If that's true, I don't think we need to add an extra element (especially since we'd need to visually hide it in this case) to label the nav.

Here's what I read:
https://tink.uk/the-difference-between-aria-label-and-aria-labelledby/

Reply:
Screen reader users often jump around the page using landmarks, such as navs, so a clear label on the nav will help give them more context/distinguish breadcrumb nav from the main menu nav, let's say. 

aria-label would suffice in this instance! It's easier to read in the code and it is actually supported better across assistive technology.

aria-label support: [https://a11ysupport.io/tech/aria/aria-label_attribute](https://a11ysupport.io/tech/aria/aria-label_attribute) 
aria-labelledby support: [https://a11ysupport.io/tech/aria/aria-labelledby_attribute](https://a11ysupport.io/tech/aria/aria-labelledby_attribute)

Reply:
Thanks for the clarification, Kara!